zb-classifier.js

```diff
@@ -1778,13 +1802,14 @@ class ZigbeeClassifier {
       this.initSeSmartPlug(node, seMeteringEndpoint, genLevelCtrlEndpoint);
     } else if (genLevelCtrlEndpoint) {
       this.initMultiLevelSwitch(node, genLevelCtrlEndpoint, lightLinkEndpoint);
-    } else if (genOnOffEndpoint) {
-      this.initOnOffSwitch(node, genOnOffEndpoint);
-    } else if (genLevelCtrlOutputEndpoint) {
-      this.initMultiLevelButton(node, genLevelCtrlOutputEndpoint,
-                                genOnOffOutputEndpoint);
-    } else if (genOnOffOutputEndpoint) {
-      this.initOnOffButton(node, genOnOffOutputEndpoint);
+    } else if (genOnOffEndpoints.length > 0) {
+      this.initOnOffSwitches(node, genOnOffEndpoints);
+    } else if (genLevelCtrlOutputEndpoints.length > 0 &&
+      genOnOffOutputEndpoints.length > 0) {
```

Comment:
nit: spacing should align with the previous line

Reply:
This still applies, inside the `if` block conditions.

Reply:
forgot about this one. fixed now.